assets/sass/components/dashboard/_googlesitekit-subtle-notification.scss

```diff
@@ -60,7 +60,10 @@
 	}
 
 	.mdc-button {
+		font-weight: $fw-medium;
 		margin: 0 auto;
+		min-height: 32px;
+		padding: 6px 16px;
```

Comment:
Noting that the primary CTA button is the shorter version, it follows the tertiary "Got it" CTA should also be, therefore this styling can be applied across the board to `.mdc_button` within the component (hence also the VRT update, which seems fine as we didn't have a mobile design for reference in the first place, and there's no meaningful difference to the wider viewport versions).

Note that both buttons are also `font-weight: 500`, the "Got it" tertiary CTA already having this weight.